src/components/text-editor/prosemirror-adapter/prosemirror-adapter.tsx

```
@@ -287,6 +311,10 @@ export class ProsemirrorAdapter {
         return newItem;
     };
 
+    private translator = (key: string, params?: {}): string => {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)